Sources/Common/Service/HttpClient.swift

```diff
@@ -166,9 +168,12 @@ extension CIOHttpClient {
      * Otherwise will return
      * `Customer.io iOS Client/1.0.0-alpha.16`
      */
-    static func getUserAgent(deviceInfo: DeviceInfo) -> String {
-        var userAgent = "Customer.io iOS Client/"
-        userAgent += deviceInfo.sdkVersion
+    static func getUserAgent(deviceInfo: DeviceInfo, sdkWrapperConfig: SdkWrapperConfig?) -> String {
+        var userAgent = "Customer.io iOS Client/\(deviceInfo.sdkVersion)"
```

Comment:
If we use enum as explained [here](https://github.com/customerio/customerio-ios/pull/174/files#r907806258), then we can directly write the initial value for `userAgent` as :

`var userAgent = "Customer.io \(sdkWrapperConfig.name.rawValue) Client/"`

Since, `Customer.io `and `Client` is same in all cases, only the source of SDK changes followed by the version of the either wrapper or the native SDK.